### ISSUE TITLE
TypeScript: Prettier adds semicolon to end of abstract class when marked as default export in definition.

### ISSUE THREAD

**Issue by joslarson** [prettier/prettier#1926]
```ts
export default abstract class NoSemicolonPlease {}
```

outputs with an unneeded semicolon:

```ts
export default abstract class NoSemicolonPlease {};
```

Works fine when removing `default` or `abstract` from the statement.

https://prettier.github.io/prettier/#%7B%22content%22%3A%22export%20default%20abstract%20class%20NoSemicolonPlease%20%7B%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22typescript%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D

**Comment by vjeux** [comment]
I just published 1.4.2 with this fix.

**Comment by joslarson** [comment]
Very nice! Thanks.
